### PR TITLE
feat(plugin): add support for the sink plug-in of the influx V2 version

### PR DESCRIPTION
**Pull request by elpsyr**
Signed-off-by: crui <crui@relaper.com>

### CODE REVIEW

File: docs/en_US/rules/sinks/plugin/influx_v2.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+### /tmp/influxRule.txt
+```json
+{
+  "id": "influx",
+  "sql": "SELECT * from  demo_stream where temperature > 50",
+  "actions": [
+    {
+      "log": {},
+      "influx":{
```
Comment (ngjaying) [inline docs/en_US/rules/sinks/plugin/influx_v2.md:78]: Why it is still influx?

File: docs/en_US/rules/sinks/plugin/influx_v2.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# InfluxDB Sink
+
+The sink will publish the result into a InfluxDB.
```
Comment (ngjaying) [inline docs/en_US/rules/sinks/plugin/influx_v2.md:3]: Should specify the InfluxDB version

File: docs/en_US/rules/sinks/plugin/influx_v2.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+## Compile & deploy plugin
+
+Please make following update before compile the plugin,
```
Comment (ngjaying) [inline docs/en_US/rules/sinks/plugin/influx_v2.md:7]: @superrxan Please review the compile information here and help to add the CI to build this plugin

**Review by ngjaying — CHANGES_REQUESTED** [review]

**Comment by ngjaying** [comment]
@elpsyr @superrxan Is it better to name it influx2 or influxV2? Go convention is to MixedCaps as variable instead of underscore, [see this](https://go.dev/doc/effective_go#mixed-caps)

File: extensions/sinks/influx_v2/influx_v2.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+	"about": {
+		"trial": true,
+		"author": {
+			"name": "elpsyr",
+			"email": "hellccqcq@gmail.com",
+			"company": "",
+			"website": "https://github.com/elpsyr"
+		},
+		"helpUrl": {
+			"en_US": "https://github.com/lf-edge/ekuiper/blob/master/docs/en_US/rules/sinks/plugin/influx.md",
```
[the remark this answers is not in the document]
Comment (elpsyr) [reply extensions/sinks/influx_v2/influx_v2.json:11]: Yes, I will correct this URL.

**Comment by superrxan** [comment]
to build this as an official release plugin 
should add build target in Makefile ``sinks/influx_v2`` where ``sinks`` is sinks folder and ``influx_v2`` is influx_v2 source code folder
if this section
``` shell
PLUGINS := sinks/file \
	sinks/influx \
	sinks/zmq \
	sinks/image \
	sinks/redis \
	sinks/sql   \
	sources/random \
	sources/zmq \
	sources/sql \
	sinks/tdengine \
	functions/accumulateWordCount \
	functions/countPlusOne \
	functions/image \
	functions/geohash \
	functions/echo \
	functions/labelImage
```

File: docs/en_US/rules/sinks/plugin/influx_v2.md (hunk 3 above)
Comment (elpsyr) [reply docs/en_US/rules/sinks/plugin/influx_v2.md:7]: @superrxan Thank you for your reply. I will add information about the plug-in in makefile later :)

Comment (elpsyr) [reply docs/en_US/rules/sinks/plugin/influx_v2.md:7]: @ngjaying After discussion, adding CI requires extensions Mod modification will be added in subsequent pr

**Review by ngjaying — APPROVED** [review]

**Pull request merged**